### ISSUE TITLE
Unable to reproduce the results in the paper

### ISSUE THREAD

**Issue by lilong-wen** [tdemin16/Continual-LayerNorm-Tuning#1]
I cloned your code and ran the experiment without changing any parameters using the bash script file `train_two.sh.` The results obtained were lower than those reported in the paper. Are there any parameters that need to be adjusted? 

here are the final results on imagnet-r:
`[Average accuracy till task10] Acc@1: 72.7249 Acc@5: 89.2329 Acc_Task: 0.3051 Loss: 1.3549 Forgetting: 12.8362 Backward: -12.8362`

and here are the final results on cifar100:
`[Average accuracy till task10]  Acc@1: 81.4500  Acc@5: 97.8800  Acc_Task: 0.2876        Loss: 0.6245    Forgetting: 14.0111     Backward: -14.0111`

I'm using a single Tesla V100 GPU while running these two experiments.

**Comment by tdemin16** [comment]
Hi, thanks for taking an interest in our work. The code should be self-contained, so you should not change anything.
The experiments in the table have been obtained using an RTXA5000 for all approaches based on parameter-efficient fine-tuning (during my experiments, I used a TitanRTX as well).
Please consider that the number I reported in the table has been averaged across three runs with seeds 0, 1, and 2 (the same seed has been used for CODA-P, DualPrompt, and L2P).
I just re-run the approach using an RTX4090 (seed 0), and it is giving me
```[Average accuracy till task10]  Acc@1: 75.6517  Acc@5: 90.6553  Acc_Task: 0.3205        Loss: 1.0682    Forgetting: 9.1946      Backward: -9.1775```

If it can help you, I can provide you
* My pre-trained weights (you can evaluate using your GPU);
* My conda environment (maybe there is something off there);
* The code I am running.

Also, if you can, try with an RTX GPU (I used only those), although the gap is big maybe that is the problem.

Let me know if I can help further!

**Comment by lilong-wen** [comment]
Thanks for your reply, I will re-run these experiments based on your suggestions.
